modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Models/PublisherExtensions.cs

```diff
@@ -70,5 +70,19 @@ public static class PublisherExtensions {
                 StatusMessage = model
             };
         }
+
+        /// <summary>
+        /// Create an api model from service model
+        /// </summary>
+        public static ConfiguredEndpointsResponseApiModel ToConfiguredEndpointsApiModel(
```

Comment:
Can the method be called `ToApiModel` to be consistent with the rest of the transformation calls in the controller ?

Reply:
done